app/assets/javascripts/order_option_assignments.coffee

```diff
@@ -146,4 +146,6 @@ $(document).ready ->
 
     # select all of the items in the "to" list before submitting
     $('#new-order-option-assignment-form').submit ->
-      $('#collectionsChooser_toList option').prop('selected', true)
\ No newline at end of file
+      $('#collectionsChooser_toList option').prop('selected', true)
+   
+  $(document).tooltip()
```

Comment:
move to a new script, tooltip.coffee

Reply:
maybe also add some comments to that new file, explaining what it is doing